src/utils.ts

```diff
@@ -36,12 +38,24 @@ export function formatAddress(address: Address, lang: string): string {
   return `${streetName} ${streetNumber}, ${postalCode} ${city}`;
 }
 
+export function formatAddresses(permits: Permit[], lang: string): string {
+  const addresses: string[] = [];
+  permits.forEach(permit => {
+    if (permit?.address) {
+      addresses.push(formatAddress(permit.address, lang));
+    }
+  });
+  return addresses.length > 0 ? addresses.join(', ') : '-';
+}
+
```

Comment:
The function signature looks a bit confusing. It's supposed to format "addresses", but we are passing in permits here. IMO, it would be better to collect the addresses from permits in the outer function and let this function do formatting only.